build/xwalk.gni

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+# This file contains Crosswalk-feature-related build flags. Users should
+# import it into their build folder by run:
+#   echo 'import("//xwalk/build/xwalk.gni")' > out/your_folder/args.gn'
+# Then run:
+#   gn gen out/your_folder
+
+# Set it false when building content_shell.
+declare_args() {
```

Comment:
It looks like you didn't read my comment in crosswalk-project/chromium-crosswalk#370 about using `build/secondary` in Chromium to get rid of the `is_build_crosswalk` hack.